mathics/builtin/specialfns/gamma.py

```diff
@@ -332,7 +332,6 @@ class Gamma(MPMathMultiFunction):
 
     rules = {
         "Gamma[z_, x0_, x1_]": "Gamma[z, x0] - Gamma[z, x1]",
-        "Gamma[1 + z_]": "z!",
```

Comment:
The rule should be
"Gamma[z_Integer]": "(z-1)!"

Reply:
Does this cause any observable difference in output?  If not, I'd recommend removing the rule unless there is good reason to add it.

Reply:
We have different implementations for Gamma and Factorial, relying on different mpmath functions. Right now, mpmath function gives the same answer, so both outputs are the same.

Reply:
Great - then let's not complicate here when there is no user-observable difference.